Gems/AtomTressFX/Code/Builders/HairAssetBuilder.cpp

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Assets/HairAsset.h>
+#include <Builders/HairAssetBuilder.h>
+#include <AssetBuilderSDK/SerializationDependencies.h>
+
+#include <AzCore/IO/FileIO.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        namespace Hair
+        {
+            void HairAssetBuilder::RegisterBuilder()
+            {
+                // Setup the builder descriptor
+                AssetBuilderSDK::AssetBuilderDesc builderDesc;
+                builderDesc.m_name = "HairAssetBuilder";
+
+                builderDesc.m_patterns.emplace_back(AssetBuilderSDK::AssetBuilderPattern(
+                    AZStd::string::format("*.%s", AMD::TFXFileExtension), AssetBuilderSDK::AssetBuilderPattern::PatternType::Wildcard));
+                builderDesc.m_busId = azrtti_typeid<HairAssetBuilder>();
+                builderDesc.m_version = 3;
+                builderDesc.m_createJobFunction =
+                    AZStd::bind(&HairAssetBuilder::CreateJobs, this, AZStd::placeholders::_1, AZStd::placeholders::_2);
+                builderDesc.m_processJobFunction =
+                    AZStd::bind(&HairAssetBuilder::ProcessJob, this, AZStd::placeholders::_1, AZStd::placeholders::_2);
+
```

Comment:
while this is not incorrect, we no longer need to use bind for anything now that we have lambdas.
```
eg, createJobFUnction = [this](..) { HairAssetBuilder::CreateJobs(...) }

```

Reply:
I'd leave it for now as this CR is massive and it doesn't seem to be high priority.  Let me know if you want to re-open.
Adding @amzn-rhhong for future visibility